exist-core/src/main/java/org/exist/collections/triggers/XQueryTrigger.java

```diff
@@ -71,7 +72,7 @@
 */
 public class XQueryTrigger extends SAXTrigger implements DocumentTrigger, CollectionTrigger {
 
-    protected Logger LOG = LogManager.getLogger(getClass());
+    private static final Logger LOG = LogManager.getLogger(XQueryTrigger.class);
```

Comment:
LogManager.getLogger() works as well no?

Reply:
Yes they are equivalent, the old version requires an unnecessary function call, whereas the second can be done statically by the compiler.